omniauth-greenhouse.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = %w(lib)
 
   spec.add_development_dependency 'bundler', '~> 1.3'
-  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rake', '~> 0'
 
-  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1.1.1'
-  spec.add_runtime_dependency 'omniauth', '~> 1.2.1'
+  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1'
+  spec.add_runtime_dependency 'omniauth', '~> 1.2.2'
```

Comment:
We can actually remove this guy I think.  the `omniauth-oauth2` gem includes this dependency:

https://rubygems.org/gems/omniauth-oauth2/versions/1.4.0